ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/index.scss

```
@@ -0,0 +1,22 @@
+.advanced-gas-fee-input {
+  a.advanced-gas-fee-input__edit-link {
+    display: inline;
+    font-size: 12px;
+    padding: 0;
+    white-space: nowrap;
+  }
+
+  .form-field__heading-title > h6 {
+    font-size: 12px;
+  }
+
+  &__border {
+    padding-bottom: 20px;
```

[Should adhere to 8px grid](https://consensys.slack.com/archives/G1L7H42BT/p1637778098226300) 
```suggestion
    padding-bottom: 16px; // or 24px
```